src/urbit/app/chess.hoon

```diff
@@ -273,13 +273,52 @@
       :-  :~  :*  %give  %fact  ~[/challenges]
                   %chess-update  !>([%challenge src.bowl challenge])
               ==
+              :*  %pass   /hark-store
+                  %agent  [our.bowl %hark-store]
+                  %poke   :-  %hark-action
+                          !>  :-  %unread-count
+                              :-  %chess
+                              :-  /challenges
+                              :-  %.y  1
+              ==
+              :*  =/  title=(list content:hark)
+                    ~[[%text (crip "{<src.bowl>} has sent a challenge")]]
```

Comment:
`~[[%ship src.bowl] [%text ' has sent a challenge']]` would also work, and is probably faster since it doesn't involve string injection and conversion from `tape` to `cord`.